### ISSUE TITLE
Missing byteorder parameter

### ISSUE THREAD

**Issue by jaroslavknotek** [denballakh/ranger-tools#11]
I am trying to read `lang.dat` using the `DAT.from_file` method. This call will subsequently fails with the error:

```
TypeError: from_bytes() missing required argument 'byteorder' (pos 2)
```

Full stack
```
Traceback (most recent call last):
  File "/home/jry/source/SRHD-lang-dat-translate/python/app.py", line 8, in <module>
    rangers.dat.DAT.from_file(dat_file_path)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/dat.py", line 311, in from_file
    return cls.from_dat(path)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/dat.py", line 263, in from_dat
    return cls.from_bytes(data, fmt=fmt)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/dat.py", line 222, in from_bytes
    ).read_bytes(data)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/std/dataclass.py", line 302, in read_bytes
    return self.read_with_memo(IBuffer(data), memo)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/std/dataclass.py", line 313, in read_with_memo
    result = self.read(buf)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/std/dataclass.py", line 973, in read
    data = self.dcls1.read(buf)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/std/dataclass.py", line 1167, in read
    data = self.bytes_xor(data1, data2)
  File "/home/jry/source/SRHD-lang-dat-translate/.venv/lib/python3.10/site-packages/rangers/std/dataclass.py", line 1145, in bytes_xor
    return (int.from_bytes(d1) ^ int.from_bytes(d2)).to_bytes(max(len(d1), len(d2)))
TypeError: from_bytes() missing required argument 'byteorder' (pos 2)
```

**Comment by denballakh** [comment]
My code works on newest CPython (3.11). I haven't tested it on older versions.
In 3.11 `byteorder` argument has default value: https://docs.python.org/3.12/library/stdtypes.html#int.from_bytes.
I will fix this issue someday to support 3.10 in this case.

**Comment by jaroslavknotek** [comment]
Thanks, I have already created a pull requests that does that. You can see it here https://github.com/denballakh/ranger-tools/pull/12
